services/espnet-tts-fr/src/app.py

```diff
@@ -51,18 +51,26 @@ def isfloat(num):
 
 def processSegment(s):
     if s.isnumeric() or isfloat(s):
-        logger.debug(f"Case: '{s}' is numeric or float.")
-        return num2words(s, lang='fr')
+        logger.debug("Case: numeric or float.")
+        s = num2words(s, lang='fr')
     elif "," in s:
-        logger.debug(f"Case: '{s}' has , as separator")
+        logger.debug("Case: has a , as separator")
```

Comment:
Hmm, does this still get run for text uses of `,`?

Reply:
Could also catch with a regexp find replace (`(\d+),(\d+)` to `$1.$2`).

Reply:
It catches text with `,`, but it won't perform any changes since on the next line, it couldn't cast the text to float. 

I will look into regex and string processing more, thanks for the suggestion! I genuinely forgot about regex and went straight for the simpliest implementation.